src/api/spec/models/project_spec.rb

```diff
@@ -288,7 +288,7 @@
     end
   end
 
-  describe '#deleted?' do
+  describe '.deleted?' do
```

Comment:
@bgeuken : @mdeniz said it should be a `.` instead of `#`. Would make sense actually since it's static.